src/recensio/plone/browser/review.py

```diff
@@ -56,7 +59,9 @@ def get_canonical_url(self):
 
     def get_metadata_review_author(self):
         return get_formatted_names(
-            " <br/> ", ", ", self.context.reviewAuthors, lastname_first=True
+            self.context.reviewAuthors,
+            lastname_first=True,
+            full_name_separator=" <br/> ",
```

Comment:
This is not the code that I see in the old system
Can you explain why?

Reply:
I changed the signature because I don't see why we should always pass the separators just because one or two calls need a different one.

Reply:
Let'\s do this improvement at the end

Reply:
I'll keep that in mind for next time.

Reply:
:roll_eyes: